server/webhook_parser.go

```diff
@@ -214,8 +214,17 @@ func parseWebhookCommentCreated(jwh *JiraWebhook) (Webhook, error) {
 		text:        truncate(jwh.Comment.Body, 3000),
 	}
 
+	attachCommentNotifications(wh)
+
+	return wh, nil
+}
+
+func attachCommentNotifications(wh *webhook) {
```

Comment:
nit/naming: `s/attach/append/`? or even `parseCommentNotifications`? I got confused by `attach` for a second, expecting that to mean "post" or "reply", but saw no `api` parameter.

Reply:
@levb good point, changed to append (also made the change for the other attachNotifications function